src/app/App.jsx

```diff
@@ -21,11 +15,17 @@ function App() {
     <Wrapper>
       <AppHeader user={user} handleLogoutStatus={setUser} />
       <Routes>
-        <Route path="/" element={<VideoList />} />
+        <Route path="/" element={<Main />} />
         <Route path="/login" element={<Login handleLoginStatus={setUser} />} />
```

Comment:
현재 Login component에 onLogin이라는 prop 전달되고 있는데 App Component에서 넘기는게 없는 것 같네요.  
아래와 같이 전달해줘될 것 같네요.  prop 명은 그대로 적어봤습니다.
```suggestion
        <Route path="/login" element={<Login onLogin={onLogin} handleLoginStatus={setUser} />} />
```